auction-app-frontend/src/pages/Home/Home.jsx

```diff
@@ -1,7 +1,136 @@
-import { Link } from "react-router-dom";
+import { useState, useEffect } from "react";
+
+import {
+  ProductGrid,
+  ErrorComponent,
+  LoadingComponent,
+  Button,
+} from "src/components";
+
+import ProductService from "src/services/productService";
+import CategoryService from "src/services/categoryService";
+
+import { go } from "src/assets/icons";
+
+import "./style.scss";
 
 const Home = () => {
-  return <div>Home Page</div>;
+  const [activeTab, setActiveTab] = useState("newArrivals");
+  const [page, setPage] = useState(0);
+  const [items, setItems] = useState([]);
+  const [hasMore, setHasMore] = useState(true);
+
+  const {
+    product,
+    loading: productLoading,
+    error: productError,
+  } = ProductService.useProductRandom();
+  const {
+    categories,
+    loading: categoriesLoading,
+    error: categoriesError,
+  } = CategoryService.useCategoriesTopLevel();
+
+  // determine fetching function based on activeTab
+  const endpoint = activeTab === "newArrivals" ? "newArrivals" : "lastChance";
+  const {
+    data: products,
+    loading: productsLoading,
+    error: productsError,
+  } = ProductService.useProductsPaginated(endpoint, page, 8);
+
+  useEffect(() => {
+    const updatedItems = [
+      ...items,
+      ...products.filter(
```

Comment:
We shouldn't be doing this, page should return date only for that page, so there shouldn't be any duplicats